addons/stock/report/report_stockpicking_operations.xml

```diff
@@ -153,6 +151,25 @@
                                     </tr>
                               </tbody>
                             </table>
+                            <table class="table table-condensed" t-if="o.package_ids and o.picking_type_entire_packs">
+                                <thead>
+                                    <tr>
+                                        <th width="25%">Package</th>
+                                        <th width="25%" class="text-center">Barcode</th>
+                                        <th width="25%" class="text-left">Source</th>
+                                        <th width="25%" class="text-right">Destination</th>
+                                    </tr>
+                                </thead>
+                                <tbody>
+                                    <tr t-foreach="o.package_ids.sorted(key=lambda p: p.name)" t-as="package">
+                                        <t t-set="package" t-value="package.with_context({'picking_id':o.id})" />
+                                        <td><span t-field="package.name"/></td>
+                                        <td><img t-att-src="'/report/barcode/?type=%s&amp;value=%s&amp;width=%s&amp;height=%s' % ('Code128', package.name, 600, 100)" style="width:100%;height:50px"/></td>
```

Comment:
width 100%? usually we set it to half the pixels we pass to the route, so 300px in this case